build/ios/scripts/build.sh

```diff
@@ -12,9 +12,9 @@ echo "Building for this architectures: $ARCHS"
 
 for ARCH in ${ARCHS}; do
     if [ "${ARCH}" == "i386" ] || [ "${ARCH}" == "x86_64" ]; then
-        PLATFORM="iPhoneSimulator"
+        PLATFORM="iphonesimulator"
```

Comment:
For @bassosimone: this is not just a variable renaming, we need lower case values to pass it to xcrun.
The name with CamelCase was used to retrieve the SDK directly with the folder, but I think using xcrun is more robust!